lib/plurimath/math/function/binary_function.rb

```diff
@@ -56,12 +56,32 @@ def to_omml_without_math_tag
           [r_tag]
         end
 
+        def to_math_zone(spacing, last = false, indent = true)
+          constant = self.class::FUNCTION
+          puts "#{spacing}\"#{to_asciimath}\" #{constant[:name]}"
```

Comment:
Instead of `to_asciimath`, it should take the arguments from `to_display`, such as `to_display(type: asciimath)`, `to_display(type: mathml)`

Reply:
@ronaldtse You mentioned **MathML** as an example type, I just want to clarify do we need to support it or is it just an example?
If we are indeed supporting **MathML**, how do you suggest we display it? Something that I have in mind is like below,


```
 └─Math zone
   └─ “<math display="block" xmlns="http://www.w3.org/1998/Math/MathML"><mfrac><mn>1</mn><mrow><mn>2</mn><mi>π</mi></mrow></mfrac></math>”
     └─ “<mfrac><mn>1</mn><mrow><mn>2</mn><mi>π</mi></mrow></mfrac>” fraction
        ├─ “<mn>1</mn>” numerator
        └─ “<mrow><mn>2</mn><mi>π</mi></mrow>” denominator

```

Reply:
Absolutely, I think this is a great approach. Let’s do this so we have a generic math display tree function.